rozy/src/installers/conda.rs

```diff
@@ -26,8 +27,9 @@ pub fn conda_install(
     delete_if_exists(to_dir)?;
 
     let conda_cache_dir = tempdir()?;
-    let mut command = std::process::Command::new(conda_bin);
+    let mut command = Command::new(conda_bin);
```

Comment:
I wonder if we should wrap this common pattern of "make a new command with stdout->stderr" ?)